protostar/cairo/cairo_bindings.py

```diff
@@ -21,27 +32,40 @@ def call_sierra_to_casm_compiler(
 
 
 def call_cairo_to_casm_compiler(
-    input_path: Path, output_path: Optional[Path] = None
+    input_path: Path,
+    output_path: Optional[Path] = None,
+    cairo_paths: Optional[list[Path]] = None,
```

Comment:
nitpick: I believe this should be a singular `cairo_path` since it's how cairo lang defines it (did it change with cairo 1.0?)

Reply:
Interesting, so we don't want to handle a situation when there are multiple libs in different locations? I mean, I added this myself (the plural here) as I thought it might come useful.

Reply:
Also, I don't see how it could work with nested dependencies (https://github.com/software-mansion/protostar/pull/1433#discussion_r1105672811) but maybe I'm missing the idea here.

Reply:
@karol-bisztyga it is name inspired by unix PATH, what @Arcticae suggested is to keep the type and only rename the variable to `cairo_path`, does it make sense?

Reply:
Totally, I misunderstood, thanks for explaining!

Reply:
[cairo_paths => cairo_path](https://github.com/software-mansion/protostar/pull/1433/commits/740f54128a2c19e76e25139ff9ef64ddf78a36e2)

Reply:
Yep that's what i meant :)